packages/open-truss/src/configuration/engine-v1/RenderConfig.tsx

```diff
@@ -48,7 +52,14 @@ export function RenderConfig({
     configPath,
     COMPONENTS,
   ) as FrameWrapper
+
+  const signals = createSignals(config.signals)
+  const propErrors = validateComponentProps(config.frames, COMPONENTS, signals)
```

Comment:
Made a validation function for component props. Right now this is console logging any errors. We don't want to throw any errors so it doesn't break the config builder, which has a valid use case for "invalid component props".

Reply:
I really like this error validation check! If we move this into `<Frame />`'s new `try ... catch` then it would be totally fine to throw errors as that code catches them and renders them to the user. What do you think about doing that?

Reply:
I _think_ we want both? Soon when we'll be rendering frames selectively for multi-stage workflows, this creates a scenario where code paths for sub frames won't be exercised until the workflow is run. In that world, we want this validation where it is since it checks every frame regardless of if it is rendered or not, allowing us to confirm the validity of a workflow before it is run.

That being said, we probably also want validation at the Frame level as well since that will likely have more "runtime" data and we can perform validations we can't otherwise at config "compile" time.

Reply:
Having it in both places makes sense. Could you move it into the `if (validateConfig)` block I added recently (a few dozen up)? I'm using that in the config builder for when we are okay with a config being wrong